pkg/newlog/cmd/newlogd.go

```diff
@@ -745,11 +755,14 @@ func writelogFile(logChan <-chan inputEntry, moveChan chan fileChanInfo) {
 	for {
 		select {
 		case <-checklogTimer.C:
+			domainUUIDLock.Lock()
 			timeIdx++
 			checkLogTimeExpire(fileinfo, &devStats, moveChan)
-			checklogTimer = time.NewTimer(5 * time.Second)  // check the file time limit every 5 seconds
+			checklogTimer = time.NewTimer(5 * time.Second) // check the file time limit every 5 seconds
+			domainUUIDLock.Unlock()
```

Comment:
is this domainUUIDLock needed in this block?

Reply:
We modify domainUUID inside checkLogTimeExpire, so yes (parallel access may come from handleDomainStatus...).

Reply:
I only found we inside access the appStatsMap, maybe that needs to be a separate lock?

Reply:
I mean this line: https://github.com/lf-edge/eve/blob/0aa67ad9ee6f509061d828aab602bd8a63dab59a/pkg/newlog/cmd/newlogd.go#L1303
I can see that appStatsMap modified only inside writeLogFile goroutine's functions

Reply:
Ok. Sorry i was using an old workspace, which didn't have this code merged in 3 weeks ago. thanks.